bitwarden_license/src/CommCore/Services/ProviderService.cs

```diff
@@ -419,6 +422,15 @@ public async Task RemoveOrganization(Guid providerId, Guid providerOrganizationI
             await _eventService.LogProviderOrganizationEventAsync(providerOrganization, EventType.ProviderOrganization_Removed);
         }
 
+        public async Task LogProviderAccessToOrganization(Guid organizationId)
+        {
+            var providerOrganization = await _providerOrganizationRepository.GetByOrganizationId(organizationId);
+            var organization = await _organizationRepository.GetByIdAsync(organizationId);
+            await _eventService.LogProviderOrganizationEventAsync(providerOrganization, EventType.ProviderOrganization_VaultAccessed);
+            await _eventService.LogOrganizationEventAsync(organization, EventType.Organization_VaultAccessed);
```

Comment:
There is no null-checking for `providerOrganization` or `organization`. If for any reason null gets passed to one of the _eventService.LogXXX methods they would throw a NullRefEx. Unfortunately a common pattern in the EventService. We might just add an early return when the main param used for the EventLog is Null